app/views/layouts/mailer.text.erb

```diff
@@ -11,16 +11,14 @@ Archive of Our Own
 <%= yield(:footer_note) %>
 
 <% end %>
-If you've received this message in error, please contact Support at <%= root_url %>support.
-
-The Archive of Our Own is a fan-run and fan-supported archive that relies on your donations: <%= donate_url %>.
+<%= t 'mailer.general.footer.general.text', support_url: root_url+"support", donate_url: donate_url %>
 <% if content_for?(:sent_at) %>
 
-Sent at: <%= yield(:sent_at) %>.
+<%= t 'mailer.general.footer.sentat' %>  <%= yield(:sent_at) %>.
 
 <% end %>
 <% if content_for?(:sent_by) %>
 
-Sent by: <%= yield(:sent_by) %>.
+<%= t 'mailer.general.footer.sentby'%>  <%= yield(:sent_by) %>.
```

Comment:
Same as above regarding double spaces and incorporating the full stop into the interpolated string.